app/src/main/res/layout/fragment_first.xml

```diff
@@ -34,7 +34,7 @@
     <EditText
         android:id="@+id/input_amount"
         android:layout_width="200dip"
-        android:text="5.00"
+        android:text=""
```

Comment:
Ok to remove this value here but would it be okay to set the initial "AMOUNT_DEFAULT" for the new SpecialAmount-DropDown? :) This would retain the current app behaviour that starts off with a default payment amount. :)